app/components/navigation/breadcrumb_component.rb

```diff
@@ -0,0 +1,9 @@
+module Navigation
+  class BreadcrumbComponent < ViewComponent::Base
+    attr_accessor :under_hero
+
+    def initialize(under_hero: false)
```

Comment:
As I understand it, this is just to allow configuring the theming - two options I can see

1) We require it to be manually configured, and there with be other scenarios for the alternate styling - so it would be better with a name that reflects what it does, not why its used
2) We only use it for under heros - in which case can this just be handled with css selectors? `.this-class + .that-class .breadcrumb` ?

If possible I think (2) is the better way to go

Reply:
Yeah that's a far better idea 👍